types/reactjs-localstorage/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for reactjs-localstorage 1.0
+// Project: https://github.com/null-none/react-localstorage
+// Definitions by: Rick Wilson <https://github.com/rwilson504>
+//                 Dan Cox <https://github.com/powerappsdev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace reactLocalStorage;
```

Comment:
https://www.npmjs.com/package/reactjs-localstorage doesnt say it can be used as global. It either needs to be removed or if this can be used as global, add test for global.
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.

Reply:
@sheetalkamat  Thank you for your help, this is my first time writing a d.ts file.